docker-compose.yaml

```diff
@@ -24,12 +24,16 @@ services:
       DEBT_TOPIC_ADDRESS: ${DEBT_TOPIC_ADDRESS:-ffc-pay-debt-data}${MESSAGE_QUEUE_SUFFIX}
       QC_TOPIC_ADDRESS: ${QC_TOPIC_ADDRESS:-ffc-pay-quality-check}${MESSAGE_QUEUE_SUFFIX}
```

Comment:
These names are tidied up in a subticket elsewhere but for this PR to work, QC needs to be `ffc-pay-debt-data-response`